RestaurantList.js

```diff
@@ -3,33 +3,58 @@ import Restaurant from './Restaurant'
 import RestaurantModel from './RestaurantModel'
 
 class RestaurantList extends React.Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.displayName = "RestaurantList"
-
     this.state = {
-      restaurants : [
-        new RestaurantModel("Moses", "Burger", 2)
-        ,new RestaurantModel("המזנון", "Pita", 3)
-        ,new RestaurantModel("Fresh", "Bari", 1)
-        ,new RestaurantModel("Hachim", "Mideterian", 3)
-      ]
+      selectedRestaurant : 0
     }
   }
 
+  handleSelectedRestaurant(restaurant, i) {
+    console.log('selected restaurant ' + i)
+    this.setState({ selectedRestaurant: i });
+  }
+
   render() {
+    console.log(this.props.restaurants[this.state.selectedRestaurant].address)
     return (
       <div className="row">
         <div className="col-md-7">
-          <ul class="media-list">
-            {this.state.restaurants.map(function(restaurant, i) {
-              return <Restaurant model={restaurant} key={i} />
-            })}
+          <ul className="media-list">
+            {this.props.restaurants.map(
+              function(restaurant, i) {
+                var selected = (this.state.selectedRestaurant == i)
+                return (<Restaurant
+                  onClick={this.handleSelectedRestaurant.bind(this, restaurant, i)}
+                  model={restaurant}
+                  key={i}
+                  selected={selected} />)
+              }, this)
+            }
           </ul>
         </div>
+        <div className="col-md-5">
+          <div className="restaurant-map-display">
+            <iframe className="restaurant-map-display" height="600vh" frameborder="0" src={"https://www.google.com/maps/embed/v1/place?key=AIzaSyANf9OZ_oX04XoE0DCKSKi5CZvDuUFIzQE&q=" + this.props.restaurants[this.state.selectedRestaurant].address} allowfullscreen></iframe>
+          </div>
+        </div>
```

Comment:
too small, and not done well enough. 

Should we decide to use google maps once we will need to make a JS based integration, and make it grow / shrink by the columns / width. For now it's a hack with predefined height (which I could not get to be responsive).